query-node/substrate-query-framework/cli/src/parse/WarthogModelBuilder.ts

```diff
@@ -95,6 +97,7 @@ export class WarthogModelBuilder {
       description: o.description?.value,
       isInterface: o.kind === 'InterfaceTypeDefinition',
       interfaces: o.kind === 'ObjectTypeDefinition' ? this.getInterfaces(o) : [],
+      relatedEntityImports: new Set<string>(),
```

Comment:
It feels like this field is not a genuine property of `ObjectType` but is calculated based on the fields. This is a potentially dangerous place, as we have methods for adding new fields to the existing ObjectType, and this would likely lead to an inconsistent state. Maybe it can be replaced by a method on `WarhogModel`, smth like `WarthogModel.getReferencedTypes(o: ObjectType)` ?

Reply:
Here I am initiating the set but what you mean is to get `relatedEntityImports`. Do you mean to move it `Set` initialization to `addObjectType` method?